src/component/Timer/index.jsx

```diff
@@ -22,11 +31,43 @@ const Timer = () => {
 
     const reset = () => {
       timeOverSoundAudio.play();
+      const showNotification = () => {
+        let text = null;
+        const img = 'https://source.unsplash.com/1600x900/?nature,water;';
+        if (counter.current === 6) {
+          text = `Great job! Take a long break. You have ${PERIODS.longBrk.secs} minutes.`;
+          (() => new Notification('Pomodoro', {
+            body: text,
+            icon: img
+          }))();
+        } else if (currentPeriod.id === 'short-brk' || currentPeriod.id === 'long-brk') {
+          text = `Time to get back to work! Your next break starts in ${PERIODS.work.secs} minutes.`;
```

Comment:
Can you spot the problem with this code?
```js
`Your next break starts in ${PERIODS.work.secs} minutes.`
```

Reply:
yes, sorry I also forget to change (secs) to (mins) before I pushed my branch.